docs/source/API/core/numerics/bit-manipulation.rst

```diff
@@ -0,0 +1,102 @@
+Bit manipulation
+================
+
+.. role::cpp(code)
+    :language: cpp
+
+.. role:: strike
+    :class: strike
+
+.. _KokkosBitManipulation: https://github.com/kokkos/kokkos/blob/4.1.00/core/src/Kokkos_BitManipulation.hpp
+
+.. |KokkosBitManipulation| replace:: ``<Kokkos_BitManipulation.hpp>``
+
+.. _StandarLibraryHeaderBit: https://en.cppreference.com/w/cpp/header/bit
+
+.. |StandarLibraryHeaderBit| replace:: ``<bit>``
+
+Defined in header |KokkosBitManipulation|_ which is included from ``<Kokkos_Core.hpp>``
+
+Provides function templates from the standard library header |StandarLibraryHeaderBit|_ (since C++20).
+
+Bit manipulation function templates are defined in the ``Kokkos::`` namespace since Kokkos 4.1
+
+.. _bit_cast: https://en.cppreference.com/w/cpp/numeric/bit_cast
```

Comment:
This doesn't appear as a link.

Reply:
I know.  Don't think we can render them crossed out and had them in anticipation of adding later.  Want me to remove?

Reply:
No, I think it's good to already have the links but maybe can find another way to mark unimplemented functions.

Reply:
> maybe can find another way to mark unimplemented functions

any suggestion?